.github/workflows/tests.yml

```diff
@@ -37,17 +37,33 @@ jobs:
       - name: Install Bundler RubyGems
         run: brew install-bundler-gems --groups=sorbet
 
-      - name: Install shellcheck
-        run: brew install shellcheck
+      - name: Install linting formulae
+        run: brew install shellcheck shfmt vale
 
       - run: brew style --display-cop-names
 
       - run: brew typecheck
 
       - name: Run vale for docs linting
+        run: vale docs/
+
+      - name: Install Ruby for linting tools
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
```

Comment:
Can probably use a newer one?

Reply:
When I [did that](https://github.com/EricFromCanada/brew/actions/runs/3492589078/jobs/5846528869), it said "Your Ruby version is 2.7.6, but your Gemfile specified ~> 2.6.0"

Reply:
Setting `HOMEBREW_USE_RUBY_FROM_PATH` will fix that, but I'm thinking the better thing to do here is to checkout rubydoc.brew.sh's Gemfile and use that, since that will more align with how rubydoc generation actually works. Could invoke that Rakefile too.

Reply:
Got that to almost work. Skipping the Rakefile to have it just read the source without generating output.

Reply:
Feel free to update this Gemfile if desired 👍🏻